core/src/test/java/org/veriblock/sdk/ServiceSecurityPayloadTests.java

```diff
@@ -0,0 +1,112 @@
+// VeriBlock Blockchain Project
+// Copyright 2017-2018 VeriBlock, Inc
+// Copyright 2018-2019 Xenios SEZC
+// All rights reserved.
+// https://www.veriblock.org
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+package org.veriblock.sdk;
+
+import java.io.IOException;
+import java.security.SignatureException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.veriblock.sdk.models.AltPublication;
+import org.veriblock.sdk.models.BitcoinBlock;
+import org.veriblock.sdk.models.BlockIndex;
+import org.veriblock.sdk.models.BlockStoreException;
+import org.veriblock.sdk.models.VeriBlockBlock;
+import org.veriblock.sdk.models.VeriBlockPublication;
+import org.veriblock.sdk.models.VerificationException;
+import org.veriblock.sdk.services.SerializeDeserializeService;
+import org.veriblock.sdk.transactions.VeriBlockTransactionsAtv;
+import org.veriblock.sdk.util.ParseBlocks;
+import org.veriblock.sdk.util.Utils;
+
+public class ServiceSecurityPayloadTests {
+	
+	private static VeriBlockSecurity security;
```

Comment:
random tabs instead of spaces in multiple locations

Reply:
fixed